plugin.js

```diff
@@ -72,6 +59,30 @@ const add = async function (context) {
     after: `const store`
   })
 
+  // patch Redux/index.js
+  ignite.patchInFile(`${APP_PATH}/App/Redux/index.js`, {
+    insert: `import { persistReducer } from 'redux-persist'`,
+    after: `from 'redux'`
+  })
+  ignite.patchInFile(`${APP_PATH}/App/Redux/index.js`, {
+    insert: `import ReduxPersist from '../Config/ReduxPersist'`,
+    after: `from '../Sagas/'`
+  })
+  ignite.patchInFile(`${APP_PATH}/App/Redux/index.js`, {
+    insert: `
+    let finalReducers = reducers
```

Comment:
It looks like there are some whitespace issues here. These need to be unindented by two spaces to be formatted properly. Right now it looks like this:

![image](https://user-images.githubusercontent.com/1944151/33624482-64c406dc-d9a9-11e7-8fd8-77258d33fbad.png)

A newline at the end would be nice too. 😄